hooks/shapediver/useKeyBindings.ts

```diff
@@ -28,7 +28,8 @@ export function useKeyBindings(props: Props) {
 		const {modelStateId, screenshot} = await createModelState(
 			undefined, // <-- parameterNamesToInclude: use default according to the theme
 			undefined, // <-- parameterNamesToExclude: use default according to the theme
-			true, // <-- includeImage,
+			false, // <-- includeImage,
```

Comment:
@MajorMeerkatThe3rd I also suggest to keep `includeImage` set to `true` in this case